src/main/python/gui/gui.py

```diff
@@ -23,32 +23,37 @@ def __init__(self):
         self.geometry('600x350')
         self.title('GitUp')
         self._frame = None
-        self.switch_frame(StartingMenu)
-
+        global user   
+        try:
+            user = user_account.UserAccount("/tmp/gitup/token.txt")
+            self.switch_frame(StartingMenu)
+        except ValueError:
+            self.switch_frame(LoginWindow)
+ 
     def switch_frame(self, frame_class):
         if self._frame is not None:
             self._frame.destroy()
         self._frame = frame_class(self)
         self._frame.pack()
 
+
 class StartingMenu(tk.Frame):
     def __init__(self, master):
         tk.Frame.__init__(self, master)
         global user
         tk.Label(self, text = "GitUp").pack()
         tk.Label (self, text = "v1.0.0").pack()
-        if user == None:
-            tk.Button(self, text = "Login",
+        tk.Label(self, text = "Welcome " + user.get_login() + "!").pack()
+        tk.Button(self, text = "Logout",
                 command = lambda: master.switch_frame(LoginWindow)).pack()
-        else:
-            tk.Label(self, text = "Welcome " + user.get_login() + "!").pack()
         tk.Button(self, text = "Add Project",
                 command = lambda: master.switch_frame(ExistingProjects)).pack()
         tk.Button(self, text = "View Project",
                 command = lambda: master.switch_frame(OpenProjectMenu)).pack()
         tk.Button(self, text = "Remove Project", state = tk.DISABLED,
                 command = lambda: master.switch_frame(DeleteProjectMenu)).pack()
```

Comment:
This (DeleteProjectMenu) is not implemented yet correct?

Reply:
Correct! It will be fixed once the backend part of it has been added.